stubs/cffi/_cffi_backend.pyi

```diff
@@ -0,0 +1,286 @@
+
+@final
+class CType:
+    abi: Incomplete
+    args: Incomplete
+    cname: Incomplete
+    elements: Incomplete
+    ellipsis: Incomplete
+    fields: Incomplete
+    item: Incomplete
+    kind: Incomplete
+    length: Incomplete
+    relements: Incomplete
+    result: Incomplete
+    def __dir__(self): ...
+
+@final
+class Lib:
+    def __dir__(self): ...
+
+@final
+class _CDataBase:
+    __name__: ClassVar[str]
+    def __add__(self, other): ...
+    def __bool__(self): ...
+    def __call__(self, *args, **kwargs): ...
+    def __complex__(self): ...
+    def __delitem__(self, other): ...
+    def __dir__(self): ...
+    def __enter__(self): ...
+    def __eq__(self, other): ...
+    def __exit__(self, type, value, traceback): ...
+    def __float__(self): ...
+    def __ge__(self, other): ...
+    def __getitem__(self, index): ...
+    def __gt__(self, other): ...
+    def __hash__(self): ...
+    def __int__(self): ...
+    def __iter__(self): ...
+    def __le__(self, other): ...
+    def __len__(self): ...
+    def __lt__(self, other): ...
+    def __ne__(self, other): ...
+    def __radd__(self, other): ...
+    def __rsub__(self, other): ...
+    def __setitem__(self, index, object): ...
+    def __sub__(self, other): ...
+
+@final
+class __CDataFromBuf(_CDataBase): ...  # type: ignore[misc]
+
+@final
+class __CDataGCP(_CDataBase):  # type: ignore[misc]
+    def __del__(self, *args, **kwargs): ...
+
+@final
+class __CDataOwn(_CDataBase):  # type: ignore[misc]
+    def __delitem__(self, other): ...
+    def __getitem__(self, index): ...
+    def __len__(self): ...
+    def __setitem__(self, index, object): ...
+
+@final
+class __CDataOwnGC(__CDataOwn): ...  # type: ignore[misc]
+
+@final
+class __CData_iterator:
+    def __iter__(self): ...
+    def __next__(self): ...
+
+@final
+class __FFIGlobSupport: ...
+
+@final
+class buffer:
+    __hash__: ClassVar[None]  # type: ignore[assignment]
+    def __init__(self, *args, **kwargs) -> None: ...
+    def __delitem__(self, other): ...
+    def __eq__(self, other): ...
+    def __ge__(self, other): ...
+    def __getitem__(self, index): ...
+    def __gt__(self, other): ...
+    def __le__(self, other): ...
+    def __len__(self): ...
+    def __lt__(self, other): ...
+    def __ne__(self, other): ...
+    def __setitem__(self, index, object): ...
+
+# These aliases are to work around pyright complaints.
+# Pyright doesn't like it when a class object is defined as an alias
+# of a global object with the same name.
+_tmp_CType = CType
+_tmp_buffer = buffer
+
+class FFI:
+    CData: TypeAlias = _CDataBase
+    CType: TypeAlias = _tmp_CType
+    buffer: TypeAlias = _tmp_buffer  # noqa: Y042
+
+    class error(Exception): ...
+    NULL: ClassVar[CData]
+    RTLD_DEEPBIND: ClassVar[int]
+    RTLD_GLOBAL: ClassVar[int]
+    RTLD_LAZY: ClassVar[int]
+    RTLD_LOCAL: ClassVar[int]
+    RTLD_NODELETE: ClassVar[int]
+    RTLD_NOLOAD: ClassVar[int]
+    RTLD_NOW: ClassVar[int]
```

Comment:
Looks like some of these may also be platform-specific. On my Windows machine:

```pycon
>>> import _cffi_backend as c
>>> c.FFI.RTLD_DEEPBIND
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: type object '_cffi_backend.FFI' has no attribute 'RTLD_DEEPBIND'
>>> c.FFI.RTLD_GLOBAL
0
>>> c.FFI.RTLD_LAZY
0
>>> c.FFI.RTLD_LOCAL
0
>>> c.FFI.RTLD_NODELETE
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: type object '_cffi_backend.FFI' has no attribute 'RTLD_NODELETE'
>>> c.FFI.RTLD_NOLOAD
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: type object '_cffi_backend.FFI' has no attribute 'RTLD_NOLOAD'. Did you mean: 'RTLD_GLOBAL'?
>>> c.FFI.RTLD_NOW
0
```

Reply:
Same as above. Will need to do more research